xbmc/favourites/GUIWindowFavourites.cpp

```diff
@@ -37,9 +51,33 @@ bool CGUIWindowFavourites::OnSelect(int item)
   return true;
 }
 
-bool CGUIWindowFavourites::OnPopupMenu(int item)
+bool CGUIWindowFavourites::OnMessage(CGUIMessage& message)
 {
-  return CGUIMediaWindow::OnPopupMenu(item) && Refresh(true);
+  bool bReturn = false;
```

Comment:
Coding guidelines state to avoid the Hungarian notation. This seems to be a new variable.

Reply:
Sure, will fix.